notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<p>Some very small changes in this section!</p><p> </p><ul><li>Remove comma after "The <a href="https://www.sdss.org/surveys/apogee-2/" target="_blank">Apache Point Observatory Galactic Evolution Experiment 2 (APOGEE-2)</a>" </li><li>Second paragraph, second sentence, slight rewording: "This survey was used to search for stars with spectral lines showing the tell-tale radial velocity shifts associated with stellar binaries."</li><li>Second paragraph, third sentence, changing "binary stars" to "companion stars", to make it more general: "...eclipses of primary stars by companion stars would also be observed by TESS."</li><li>Third paragraph, spelling of "Asteroseismic" (missing the first "i"; "Asteroseismology" is correct though!)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>